test/IRGen/lto_autolink.swift

```diff
@@ -43,8 +43,4 @@ import AutolinkElfCPragma
 import AutolinkModuleMapLink
 #endif
 
-// UNSUPPORTED: OS=macosx && CPU=arm64
-// UNSUPPORTED: OS=ios && CPU=arm64e
-// UNSUPPORTED: OS=watchos && (CPU=arm64_32 || CPU=armv7k)
```

Comment:
Removing these seems to have caused failures in iOS and macOS bots, e.g., <https://ci.swift.org/job/oss-swift_tools-RA_stdlib-DA_test-device-non_executable/8901>. Is there a specific reason to believe that these are no longer necessary?

Reply:
I fixed it on linux and Android, so [I asked about fixing it on Darwin too](https://github.com/swiftlang/swift/pull/81398#discussion_r2084309026). Allan, who originally disabled it, didn't respond, but Evan ran some other CI and said it seemed to work fine.

Looking at the error, it's exactly the same as the linux issue I fixed, so I'll submit the same fix for Darwin and if you can run this same failing CI on the Darwin fix, we can finally get it running on Darwin too.

Reply:
Submitted the fix in #81695